terraform/app.tf

```diff
@@ -105,3 +105,18 @@ resource "azurerm_mssql_database" "mssql_database" {
   sku_name    = local.mssql_sku_name
   max_size_gb = local.mssql_max_size_gb
 }
+
+resource "azurerm_application_insights" "web_app_insights" {
```

Comment:
The connection string needs to be set as the `APPLICATIONINSIGHTS_CONNECTION_STRING` app setting on the function app